CHANGES.md

```diff
@@ -188,6 +188,8 @@ To be released.
     [[#759]]
  -  Fixed a bug where `BlockChain<T>` had rendered and evaluated actions in
     the genesis block during forking.  [[#763]]
+ -  Fixed a bug where transactions which were not propagated sufficiently,
+    could not be included in a block when reorg happened.  [[#775]]
```

Comment:
I believe this explanation does not sufficiently scope the situation, e.g., it is vague where's the borderline between sufficient and insufficient propagation.  How about this?

> Fixed a `Swam<T>`'s bug that some `Transaction<T>`s had become excluded from mining `Block<T>`s after reorg from α to β where a `Transaction<T>` was once included by a `Block<T>` (α) and not included by an other `Block<T>` (β) for the same `Index` due to the latency gap between nodes.